Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -289,4 +289,21 @@ extension DotLottieFile {
     }
   }
 
+  /// Loads an DotLottie from a data synchronously. Returns a `Result<DotLottieFile, Error>`
+  /// - Parameters:
+  ///   - data: The data(`Foundation.Data`) object to load DotLottie from
+  ///   - filename: The name of the lottie file without the lottie extension. eg. "StarAnimation"
+  public static func loadedFrom(
+    data: Data,
+    filename: String)
+    -> Result<DotLottieFile, Error>
```

Comment:
Rather than using `Result`, we should use `throws` and use just `DotLottieFile` as the return type

Reply:
I actually took reference from the other signatures where `Result` was used over throws. But, the feedback makes sense. Have made the changes.